uportal-war/src/main/java/org/jasig/portal/groups/pags/dao/EntityPersonAttributesGroupStore.java

```diff
@@ -317,7 +317,7 @@ private GroupDefinition initGroupDef(IPersonAttributesGroupDefinition group) {
             TestGroup tg = new TestGroup();
             Set<IPersonAttributesGroupTestDefinition> tests = testGroup.getTests();
             for(IPersonAttributesGroupTestDefinition test : tests) {
-                IPersonTester testerInst = initializeTester(test.getTesterClassName(), test.getAttributeName(), test.getTestValue());
+                IPersonTester testerInst = initializeTester(test);
                 tg.addTest(testerInst);
```

Comment:
Perhaps a try-catch block around the code that defines the test group (lines 317 to 323)?